Tests/Runtime/Native/BreadcrumbsAnrTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Backtrace.Unity.Model;
+using Backtrace.Unity.Model.Breadcrumbs;
+using Backtrace.Unity.Model.Breadcrumbs.InMemory;
+using Backtrace.Unity.Tests.Runtime.Native.Mocks;
+using NUnit.Framework;
+using System.Linq;
+using UnityEngine;
+
+namespace Backtrace.Unity.Tests.Runtime.Native
+{
+    public sealed class BreadcrumbsAnrTests
+    {
+        [Test]
+        public void TestAnrBreadcrumbReporting_ShouldAddAnrBreadcrumb_BreadcrumbSuccessfullyStored()
+        {
+            const int expectedNumberOfBreadcrumbs = 1;
+            var backtraceStorageManager = new BacktraceInMemoryLogManager();
+            var breadcrumbs = new BacktraceBreadcrumbs(backtraceStorageManager, BacktraceBreadcrumbType.System, UnityEngineLogLevel.Warning);
+            var configuration = ScriptableObject.CreateInstance<BacktraceConfiguration>();
+            var nativeClient = new TestableNativeClient(configuration, breadcrumbs);
+
+            nativeClient.SimulateAnr();
+            nativeClient.Update(0);
+
+            Assert.AreEqual(expectedNumberOfBreadcrumbs, breadcrumbs.LogManager.Length());
+            Assert.AreEqual(TestableNativeClient.AnrMessage, backtraceStorageManager.Breadcrumbs.First().Message);
+        }
+
+        [Test]
+        public void TestAnrBreadcrumbReporting_ShouldAddSingleAnrBreadcrumb_SingleBreadcrumbSuccessfullyStored()
```

Comment:
This test looks exactly the same as the previous test, do we need both?

Reply:
This test will invoke an update method multiple times. The test gonna validate if we will add breadcrumbs twice in a synchronous method.